python_modules/dagster/dagster/_core/instance/__init__.py

```diff
@@ -423,6 +426,7 @@ def get() -> "DagsterInstance":
 
         return DagsterInstance.from_config(dagster_home_path)
 
+    @public
```

Comment:
I think it would be good for @prha to weigh in on which of these DagsterInstance methods we should mark public